02_1_Setting_Up_a_Bitcoin-Core_VPS_by_Hand.md

```diff
@@ -294,9 +294,9 @@ $ source ~/.bash_profile
 
 We suggest setting up two variables to make this installation more automatic.
 
-The first variable, $BITCOIN, should be set to the current version of Bitcoin. It was 0.14.1 when we wrote this. The second will then automatically generate a truncated form used by some of the files.
+The first variable, $BITCOIN, should be set to the current version of Bitcoin. It was 0.15.1 when we wrote this. The second will then automatically generate a truncated form used by some of the files.
 ```
-$ export BITCOIN=bitcoin-core-0.14.1
+$ export BITCOIN=bitcoin-core-0.15.1
```

Comment:
@ChristopherA reported some problems with the StackScript initially when 0.15 came out. Were you able to confirm that it worked fine with your upgrades to 0.15.1?